document-store/src/integrationTest/java/org/hypertrace/core/documentstore/mongo/MongoQueryExecutorIntegrationTest.java

```diff
@@ -498,27 +497,6 @@ public void testAggregateWithMultipleGroupingLevels() throws IOException {
     assertSizeEqual(query, "mongo/multi_level_grouping_response.json");
   }
 
-  @Test
-  public void testDistinctCount() throws IOException {
-    Query query =
-        Query.builder()
-            .addSelection(
-                AggregateExpression.of(DISTINCT_COUNT, IdentifierExpression.of("quantity")),
-                "qty_count")
-            .addSelection(IdentifierExpression.of("item"))
-            .addAggregation(IdentifierExpression.of("item"))
-            .setAggregationFilter(
-                RelationalExpression.of(
-                    IdentifierExpression.of("qty_count"), LTE, ConstantExpression.of(1000)))
-            .addSort(IdentifierExpression.of("qty_count"), DESC)
-            .addSort(IdentifierExpression.of("item"), DESC)
-            .build();
-
-    Iterator<Document> resultDocs = collection.aggregate(query);
-    assertDocsEqual(resultDocs, "mongo/distinct_count_response.json");
-    assertSizeEqual(query, "mongo/distinct_count_response.json");
```

Comment:
This call is testing the `count()` API, which is missing in the new implementation.